portal-api/api-tests/v1/gef/amendments/get-amendment-url.ts

```diff
@@ -1,2 +1,4 @@
 export const getAmendmentUrl = ({ facilityId, amendmentId }: { facilityId: string; amendmentId: string }) =>
   `/v1/gef/facilities/${facilityId}/amendments/${amendmentId}`;
+
+export const putAmendmentUrl = ({ facilityId }: { facilityId: string }) => `/v1/gef/facilities/${facilityId}/amendments`;
```

Comment:
file name should be updated to reflect this :)

Reply:
Renamed, thanks!